frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/TextInput.js

```diff
@@ -146,7 +146,7 @@ function DatePicker(props) {
 function getDDMMYYYY(date) {
   if (!date) return "";
 
-  return new Date(date)?.toLocaleString("en-In")?.split(",")?.[0];
+  return new Date(date).toLocaleString("en-In").split(",")[0];
```

Comment:
@anilsingha-eGov Is en-In okay or we need to change it to en-IN

Reply:
You can ignore this one..since it is a core component change @rahuldevgarg